docs/reference/user/rich-info.md

```diff
@@ -0,0 +1,103 @@
+# Rich info {#RefRichInfo}
+
+_Author: Artyom Kazak_
+
+---
+
+This page describes a part of the user profile called "Rich info". The corresponding feature is called "Rich profiles".
+
+## Summary {#RefRichInfoSummary}
+
+For every team user we can store a list of key-value pairs that are displayed in the user profile. This is similar to "custom profile fields" in Slack and other enterprise messengers.
+
+Different users can have different sets of fields; there is no team-wide schema for fields. All fields are strings.
```

Comment:
add `fields are passed as an ordered list, and the order information is used for display in the client.`

or something to that end.

Reply:
Will do.